rest-api/core-rest/src/main/webapp/lib/gaffer.js

```diff
@@ -208,6 +211,27 @@ function updateDocUrl(properties) {
     });
 }
 
+function updateLogoLink(properties) {
+    updateElementWithId('logo', 'gaffer.properties.app.logo.link', properties, function(value, id) {
+        $('#' + id).attr("href", value);
+    });
+}
+
+function updateLogo(properties) {
+    updateElementWithId('logo', 'gaffer.properties.app.logo.src', properties, function(value, id) {
+        $('#' + id).prepend("<img class='logo__img' alt='swagger' height='30' width='30' src='" + value + "'/>");
```

Comment:
If you switch to version v2 in the Swagger ui, then this causes the logo to be added multiple times. We also had a problem like that with the banner, but that has since been fixed.